tests/wamr-test-suites/spec-test-script/gc_ignore_cases.patch

```diff
@@ -895,6 +906,36 @@ index 1ffd03f8..bdf7471f 100644
  (assert_return (invoke "ref") (ref.null func))
 -(assert_return (invoke "ref") (ref.null nofunc))
 -(assert_return (invoke "ref") (ref.null))
+diff --git a/test/core/return_call_ref.wast b/test/core/return_call_ref.wast
+index 353811f0..f49dc6e4 100644
+--- a/test/core/return_call_ref.wast
++++ b/test/core/return_call_ref.wast
+@@ -192,20 +192,20 @@
+ 
+ (assert_return (invoke "count" (i64.const 0)) (i64.const 0))
+ (assert_return (invoke "count" (i64.const 1000)) (i64.const 0))
+-(assert_return (invoke "count" (i64.const 1_000_000)) (i64.const 0))
++;; (assert_return (invoke "count" (i64.const 1_000_000)) (i64.const 0))
```

Comment:
Could we change to 2000 or some smaller number

Reply:
1200 works, I'll change it